routes/pointofinterest/route.js

```diff
@@ -2,26 +2,26 @@ const express = require('express');
 
 const router = express.Router();
 const connection = require('../../conf');
-const addArray = require('../../functions/Add_array');
+const transformJson = require('../../functions/transformPoiSampleJson');
```

Comment:
Keep the same name when you import your function

Reply:
corrected